docs/consume-packages/includes/restore-nuget-exe-cli.md

```diff
@@ -1,13 +1,21 @@
-Use the [restore](../../reference/cli-reference/cli-ref-restore.md) command, which downloads and installs any packages missing from the *packages* folder.
+The NuGet CLI [restore](../../reference/cli-reference/cli-ref-restore.md) command downloads and installs any packages that are missing from the packages directory.
 
-For projects migrated to PackageReference, use [msbuild -t:restore](../package-restore.md#restore-using-msbuild) to restore packages instead.
+Like `install`, the `restore` command only adds packages to disk, but doesn't change the project's dependencies. To change project dependencies, modify the *packages.config* file manually, then use `restore`.
 
-`restore` only adds packages to disk but does not change a project's dependencies. To restore project dependencies, modify `packages.config`, then use the `restore` command.
+To restore packages, run the following command:
 
-As with the other `nuget.exe` CLI commands, first open a command line and switch to the directory that contains your project file.
+```cli
+nuget restore <projectPath>
+```
+
+The `restore` command uses a solution file or a *package.config* file in the specified project path.
```

Comment:
Also tested this change? @nkolev92 to verify

Reply:
@Mikejo5000 - Yes, I tested, but I'm not completely clear whether/when it looks for a .sln file first, as opposed to a package.config file. I can test further.

Reply:
It's solution first.